### ISSUE TITLE
Tests dont jive with the readme

### ISSUE THREAD

**Issue by joshuabamboo** [learn-co-curriculum/ttt-7-valid-move#22]
Readme instructs students to convert user input to an index:

```
Arrays are indexed starting at 0 and a user playing your game is unlikely to know that. When a user types in that they'd like to fill position 1, they really mean that they'd like to fill the board array at the index of 0. You'll have to account for this in your method.
```

But the[ tests pass the index directly](https://github.com/learn-co-curriculum/ttt-7-valid-move/blob/master/spec/valid_move_spec.rb#L7) in:

```
 it 'returns true for a valid position on an empty board' do
    board = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
    index = 0
    expect(valid_move?(board, index)).to be_truthy
  end
```

If the student converts a user input of 0 to an index, then `index` will be -1 and will be a falsey value, not truthy.

Related commit: https://github.com/learn-co-students/ttt-7-valid-move-q-000/commit/bb68b5f63b04440d516483ffa413297caeedf7b1

@PeterBell

**Comment by pletcher** [comment]
Thanks for opening this issue!

Keep in mind that this space is for specific issues related to the content of this lesson.

If you're reporting on the content of this lesson, it'd be super helpful if you could include a stack trace of any errors that you're seeing.

If you'd like to report a bug about the Learn IDE, the Ask a Question form, opening a file, or the lights not working on Learn, please report a bug directly on https://learn.co. (You can find the bug report form by clicking on your avatar in the upper right, and then selecting "Report A Bug".)

Please be sure to include steps to reproduce (be as detailed as possible) and any relevant information about your environment. (Are you using the Learn IDE? Coding on a refurbished toaster?) We'll be sure to figure out what's going on — and to jump in and help — just as soon as we can. Thanks!

**Comment by PeterBell** [comment]
Hey @joshuabamboo want to take a run at fixing this?

**Comment by joshuabamboo** [comment]
@PeterBell sure thing

**Comment by gj** [comment]
Fixed in 3cb45be. Thanks @joshuabamboo !
